src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String toString() {

```

Comment:
please read about ordering of the methods.
`toString()` should be moved to the end of the class.